src/inference/src/core.cpp

```diff
@@ -16,26 +16,33 @@
 namespace ov {
 
 std::string find_plugins_xml(const std::string& xml_file) {
-    if (xml_file.empty()) {
-        const auto ov_library_path = ov::util::get_ov_lib_path();
+    std::string xml_file_name = xml_file;
+    if (xml_file_name.empty()) {
+        // Default plugin xml file name.
+        xml_file_name = "plugins.xml";
+    } else {
+        if (xml_file_name.find(util::FileTraits<char>().file_separator) != xml_file_name.npos) {
+            OPENVINO_THROW("Unsupport plugin xml relative path: ", xml_file_name.c_str());
```

Comment:
The change breaks support of absolute paths. As reference, you may take a look at `ov::util::get_plugin_path` implementation. I suppose the steps should be next:
1. If `xml_file` empty, use `"plugins.xml"`
2. If `xml_file` not with `"name.ext"` format, return as is (I guess `ov::find_plugins_xml()` should be `noexcept`, non-existent file path will be processed by pugixml implementation)
3. Search in OV folder

Reply:
Yes, I have ignored absolution path issue, will update. We have aligned that the searching folder is OV folder.
But I have 2 different points:

1. ov::find_plugins_xml() doesn't have `noexcept` modifier, so we can throw exception
2. If xml_file not with "name.ext" format, we can consider it as an attacking and throw exception.

Reply:
I guess, we should allow users to set any path they want, and it is possible and doesn't make solution insecure (but users , as previously, should guarantee path is from secure place)

Secondly, if users want to use `plugins.xml` from OV internal folder, they should fulfill some rules. Originally, it was just empty string, but now we may extend it with `"name.ext"` format. Anyway, it shouldn't be first priority over relative path (e.g. if user specify `"plugins.xml"`, and it exists in `"CWD/plugins.xml"`, this path should be used over `"OV_folder/plugins.xml"`